app/views/http_proxies/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_for @http_proxy, :url => (@http_proxy.new_record? ? http_proxies_path : http_proxy_path(:id => @http_proxy)) do |f| %>
+  <%= base_errors_for @http_proxy %>
+  <ul class="nav nav-tabs" data-tabs="tabs">
+    <li class="active"><a href="#primary" data-toggle="tab"><%= _("HTTP Proxy") %></a></li>
+
+    <% if show_location_tab? %>
+      <li><a href="#locations" data-toggle="tab"><%= _("Locations") %></a></li>
+    <% end %>
+    <% if show_organization_tab? %>
+      <li><a href="#organizations" data-toggle="tab"><%= _("Organizations") %></a></li>
+    <% end %>
+  </ul>
+
+  <div class="tab-content">
+
+    <div class="tab-pane active" id="primary">
+      <%= text_f   f, :name %>
+      <%= text_f   f, :url, :help_inline => _("URL of the proxy including schema (https://proxy.example.com)") %>
+      <%= number_f   f, :port, :help_inline => _("Port to connect to the proxy with.") %>
```

Comment:
Why is the port not part of the URL?
I think it would be easier to specify a proxy like this: `http://localhost:3128/`